Jenkinsfile

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env groovy
+/*
+ * Copyright [2018 - 2018] Confluent Inc.
+ */
+common {
+  slackChannel = '#connect-eng'
+  nodeLabel = 'docker-oraclejdk8'
```

Comment:
PS: we can use docker10 if the default docker11 is causing issues (ref: https://github.com/confluentinc/connect-plugins-common/blob/master/Jenkinsfile#L7).

Reply:
My concern with using JDK 10 or 11 is that we don't know whether the connector actually works with JDK 8. I think it's better to build connectors with older JDKs and older APIs. Ideally, we'd be able to run against multiple JDKs, but not sure how to configure that.